### ISSUE TITLE
 dataset labels

### ISSUE THREAD

**Issue by ps-star** [layumi/University1652-Baseline#58]
May I ask if the dataset contains label information? Do satellite images have latitude and longitude information

**Comment by layumi** [comment]
@ps-star 
Yes. Please check [[Latitude and Longitude]](https://drive.google.com/file/d/1PL8fVky9KZg7XESsuS5NCsYRyYAwui3S/view?usp=sharing)
